src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/SearchNavSideMenuComponentObject.java

```diff
@@ -0,0 +1,154 @@
+		searchInput.click();
+		PageObjectLogging.log("clickSearchField", "Search field was clicked", true);
+	}
+
+	public MercuryBasePageObject clickSearchSuggestion(int index) {
+		waitForElementByElement(searchSuggestions.get(index));
+		searchSuggestions.get(index).click();
+		return new MercuryBasePageObject(driver);
+	}
+
+	public void clickCancelButton() {
+		waitForElementVisibleByElement(cancelSearchCaption);
+		cancelSearchCaption.click();
+	}
+
+	public String getSearchResultHref(int searchPosition) {
+		waitForElementVisibleByElement(searchSuggestions.get(searchPosition));
+		return searchSuggestions.get(searchPosition).getAttribute("href");
+	}
+
+	public void typeInSearchField(String content, int length) {
+		waitForElementVisibleByElement(searchInput);
+		searchInput.sendKeys(content.substring(0, length));
+		PageObjectLogging.log(
+				"typeInSearchField",
+				"String >>" + content.substring(0, length) + "<< was typed in string field",
+				true
+		);
+	}
+
+	public void verifySearchSuggestionsWereVisible() {
+		waitForElementByElement(searchSuggestions.get(0));
+		Assertion.assertTrue(checkIfElementOnPage(searchSuggestions.get(0)));
+		PageObjectLogging.log("verifySearchSuggestionsWereVisible", "Search suggestions were visible", true, driver);
+	}
+
+	public void verifySearchView() {
+		Assertion.assertTrue(checkIfElementOnPage(searchView));
+		PageObjectLogging.log("verifySearchView", "Search view was displayed on full size", true, driver);
+	}
+
+	public void verifyMenuView() {
+		Assertion.assertTrue(checkIfElementOnPage(menuView));
+		PageObjectLogging.log("verifyMenuView", "Menu view was visible", true, driver);
+	}
+
+	public void verifySearchNotMatch() {
+		waitForElementByElement(searchAlerts);
+		Assertion.assertTrue(searchAlerts.getText().contains(MercuryContent.MERCURY_SEARCH_NOT_MATCH));
+	}
+
+	public void verifyOpeningArticleInNav(int anchorIndex) {
+		waitForElementVisibleByElement(noChevrons.get(anchorIndex));
+		if (noChevrons.size() != 0) {
+			noChevrons.get(anchorIndex).click();
+			PageObjectLogging.log("verifyOpeningArticleInNav", "New article opened", true, driver);
+		}
+	}
+
+	public void verifyOpeningNextLevelInNav(int anchorIndex) {
+		waitForElementVisibleByElement(chevrons.get(anchorIndex));
+		if (chevrons.size() != 0) {
+			chevrons.get(anchorIndex).click();
+			waitForElementVisibleByElement(backChevron);
+			PageObjectLogging.log("verifyOpeningNextLevelInNav", "Back button is visible", true, driver);
+		}
+	}
+
+	public void verifyBackLinkFunctionality(int anchorIndex) {
+		waitForElementVisibleByElement(chevrons.get(anchorIndex));
+		if (chevrons.size() != 0) {
+			chevrons.get(anchorIndex).click();
+			waitForElementVisibleByElement(backChevron);
+			backChevron.click();
+			PageObjectLogging.log("verifyOpeningNextLevelInNav", "Back button is working", true, driver);
+		}
+	}
+
+	public void verifyClosingNav() {
+		waitForElementVisibleByElement(overlay);
+//		tapOnElement(overlay);
```

Comment:
We are waiting for new tap gestures to be implemented and once done we will uncomment this line.